src/sentry/sentry_apps/api/endpoints/installation_external_issues.py

```diff
@@ -52,7 +53,8 @@ def post(self, request: Request, installation) -> Response:
                     project=data["project"],
                     identifier=data["identifier"],
                 ).run()
-            except Exception:
+            except Exception as e:
+                capture_exception(e)
```

Comment:
@GabeVillalobos did a cool thing with Jira where we have to show a generic useless error like something went wrong. The error we show to the user includes the error id of the Sentry error. This way user can post and say I'm seeing error xyz and debugging becomes easier. Can we do that too here and above?